recipe/meta.yaml

```diff
@@ -13,49 +11,47 @@ source:
 
 build:
   number: 0
+  #skip s390x as it lack maturin dependency
+  skip: True # [s390x]
   script:
-    {% if build_platform != target_platform %}
-    - export PYO3_CROSS_INCLUDE_DIR=$PREFIX/include
-    - export PYO3_CROSS_LIB_DIR=$PREFIX/lib/python$PY_VER
-    - export PYO3_CROSS_PYTHON_VERSION=$PY_VER
-    {% endif %}
-    - {{ PYTHON }} -m pip install . -vv
+    - {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
   build:
-    - python                                 # [build_platform != target_platform]
-    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
-    - maturin                                # [build_platform != target_platform]
-    - toml                                   # [build_platform != target_platform]
```

Comment:
How about `- rust_win-64  # [win]` in `build`?

Reply:
done